src/oc_erchef/apps/chef_objects/src/chef_cookbook_version.erl

```diff
@@ -290,10 +331,11 @@ file_list_spec() ->
 
 -spec extract_checksums(ejson_term()) -> [binary()].
 extract_checksums(CBJson) ->
+    Segments = [ <<"all_files">> | ?COOKBOOK_SEGMENTS ],
```

Comment:
We only add `all_files` to the list here because we don't want to have it treated as a segment generally (because it's not one!)